client/package.json

```diff
@@ -9,6 +9,7 @@
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.1.0",
     "redux": "^3.7.2",
+    "redux-promise": "^0.5.3",
```

Comment:
redux-promise is not being used here, even though you wanted to use it in the future you would atleast need to have `redux-thunk` :sweat_smile: